sbt-dotty/sbt-test/compilerReporter/simple/project/Reporter.scala

```diff
@@ -29,6 +29,12 @@ object Reporter {
       println(problems.toList)
       assert(problems.size == 1)
 
+      // make sure position reported by zinc are proper
+      val mainProblem = problems.head
+      val line = mainProblem.position().line() 
+      assert(line.isPresent() == true)
+      assert(line.get() == 9)
+      
       // Assert disabled because we don't currently pass positions to sbt
```

Comment:
I think that assert can be reenabled now.

Reply:
Reenabled the assert.